rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/Transaction.java

```diff
@@ -351,9 +359,10 @@ protected LRAService getLraService() {
     /**
      * return the current status of the LRA
      *
-     * @return null if the LRA is still active (not closing, cancelling, closed or cancelled
+     * @return current LRA status, will not return <code>null</code>
```

Comment:
The text after the comma is not need (and therefore confusing).
status can never be null.